### PR TITLE
Links now work

### PR DESCRIPTION
**Pull request by ms4720**
Could someone make sure I linked to the correct docs

### CODE REVIEW

File: contributors/design-proposals/architecture/namespaces.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ and reference particular entities across operations.
 A *Namespace* provides an authorization scope for accessing content associated
 with the *Namespace*.
 
-See [Authorization plugins](../admin/authorization.md)
+See [Authorization plugins](../auth/access.md)
```
Comment (cblecker) [inline contributors/design-proposals/architecture/namespaces.md]: This should be linking to https://kubernetes.io/docs/admin/authorization/

Comment (ms4720) [reply contributors/design-proposals/architecture/namespaces.md]: fixed

Comment (ms4720) [reply contributors/design-proposals/architecture/namespaces.md]: fixed

**Comment by cblecker** [comment]
@ms4720 Thanks for fixing. Please review https://github.com/kubernetes/community/pull/1309#issuecomment-340967010 about signing the CLA.

**Comment by ms4720** [comment]
CLA done

**Pull request merged**